test/box-luatest/gh_8051_set_box_cfg_thru_env_test.lua

```diff
@@ -0,0 +1,91 @@
+local server = require('luatest.server')
+local popen = require('popen')
+local clock = require('clock')
+
+local t = require('luatest')
+local g = t.group()
+
+g.after_each = function()
+    if g.server ~= nil then
+       g.server:stop()
+   end
+end
+
+local TARANTOOL_PATH = arg[-1]
+
+local function popen_run(command)
+    local cmd = TARANTOOL_PATH .. ' -i 2>&1'
+    local ph = popen.new({cmd}, {
+        shell = true,
+        setsid = true,
+        group_signal = true,
+        stdout = popen.opts.PIPE,
+        stderr = popen.opts.DEVNULL,
+        stdin = popen.opts.PIPE,
+    })
+    t.assert(ph, 'process is not up')
+
+    ph:write(command)
+
+    local output = ''
+    local time_quota = 10.0
+    local start_time = clock.monotonic()
+    while output:find('message') == nil
```

Comment:
Hmm. I don't get the `:find('message')` condition. Can you explain?

Reply:
You are right, dropped it.